src/compiler/scala/tools/nsc/typechecker/Contexts.scala

```diff
@@ -1108,10 +1108,10 @@ trait Contexts { self: Analyzer =>
       //
       // A type-import-on-demand declaration never causes any other declaration to be shadowed.
       //
-      // Scala: Bindings of different kinds have a precedence deﬁned on them:
+      // Scala: Bindings of different kinds have a precedence defined on them:
```

Comment:
Retronym has only recently switched to the web version of the spec, as he is old-school. You missed your chance to fix the typo in that commit: https://github.com/scala/scala/commit/0147c734ec97a05131d1908232e70517b7c20ab7